--- include/crt/delayimp.h
+++ include/crt/delayimp.h
@@ -202,23 +202,19 @@
 //
 ExternC
 extern
-const
 PfnDliHook   __pfnDliNotifyHook;
 
 ExternC
 extern
-const
 PfnDliHook   __pfnDliNotifyHook2;
 
 // This is the failure hook, dliNotify = {dliFailLoadLib|dliFailGetProc}
 ExternC
 extern
-const
 PfnDliHook   __pfnDliFailureHook;
 
 ExternC
 extern
-const
 PfnDliHook   __pfnDliFailureHook2;
 
 #if DELAYLOAD_VERSION >= 0x0200
